server/app/services/applicant/ApplicantService.java

```diff
@@ -398,13 +441,39 @@ CompletionStage<Application> submitApplication(
               String programName = application.getProgram().getProgramDefinition().adminName();
               notifyProgramAdmins(applicantId, programId, application.id, programName);
 
+              CompletableFuture<Void> maybeNotifyTiSubmitterFuture;
               if (tiSubmitterEmail.isPresent()) {
-                notifyTiSubmitter(tiSubmitterEmail.get(), applicantId, application.id, programName);
+                maybeNotifyTiSubmitterFuture =
+                    maybeNotifyTiSubmitter(
+                            tiSubmitterEmail.get(), applicantId, application.id, programName)
+                        .toCompletableFuture();
+              } else {
+                maybeNotifyTiSubmitterFuture = CompletableFuture.completedFuture(null);
               }
 
-              maybeNotifyApplicant(applicantId, application.id, programName);
-
-              return updateStoredFileAclsForSubmit(applicantId, programId)
+              Program applicationProgram = application.getProgram();
+              Optional<StatusDefinitions.Status> maybeDefaultStatus =
+                  applicationProgram.getDefaultStatus();
+              CompletableFuture<Void> maybeUpdateStatusAndEmailApplicantFuture;
+              if (maybeDefaultStatus.isPresent()) {
+                StatusDefinitions.Status defaultStatus = maybeDefaultStatus.get();
+                maybeUpdateStatusAndEmailApplicantFuture =
+                    CompletableFuture.allOf(
+                        setApplicationStatus(application, defaultStatus).toCompletableFuture(),
+                        maybeNotifyApplicantWithStatus(
+                                applicantId,
+                                application.getProgram().getProgramDefinition(),
```

Comment:
**suggestion:** Can swap for applicationProgram.getProgramDefinition(), since application.getProgram() is defined in that variable above.